### DISCUSSION TITLE
Prometheus for IA64 (Itanium) on HP-UX?

### DISCUSSION THREAD

**Discussion by TimothyNeto** [prometheus/prometheus#9965] [Q&A]
Hello.
Is there any efforts to compile and bundle a Prometheus package for HP-UX on IA64?
If "yes", would anyone know the title on the thread?
I have tried searching for the topic here in this GitHUB and via general Internet search. I am not finding any on first pass searches.
Thank in advance for any insight or information.
Timothy

**Comment by LeviHarrison** [comment]
Given that Go doesn't support the IA-64 instruction set (https://go.dev/doc/install/source#introduction), and therefore it isn't possible for us to compile a binary for it, I don't think so.

**Reply by LeviHarrison** [reply]
Actually, it may be possible to compile for IA-64 using [gccgo](https://gcc.gnu.org/onlinedocs/gccgo/), which is a frontend for GCC.
